packages/imperative/src/imperative/src/config/cmd/secure/secure.definition.ts

```diff
@@ -39,6 +39,13 @@ export const secureDefinition: ICommandDefinition = {
             aliases: ["p"],
             type: "boolean",
             defaultValue: false
+        },
+        {
+            name: "profile",
+            description: "Specify the profile in which you would like to configure the secure values for",
```

Comment:
Minor suggestion on wording. I think it reads better as:

Specify the profile for which you want to configure secure values